contracts/src/v0.8/llo-feeds/FeeManagerNoNative.sol

```diff
@@ -0,0 +1,479 @@
+// SPDX-License-Identifier: MIT
```

Comment:
What's the plan for this contract?

Looks like it's a copy of [FeeManager.sol](https://github.com/smartcontractkit/chainlink/blob/e315cf6e2b838390b858fbd207aa7b2cf90fac0d/contracts/src/v0.8/llo-feeds/FeeManager.sol) with some changes. I'd need to run a diff locally to figure out what are the exact changes.

Can we make this contract an explicit derivation of FeeManager.sol (not a copy)?

Reply:
> Can we make this contract an explicit derivation of FeeManager.sol (not a copy)?

This would be ideal.